File: Python/tests/test_bart_prospect_phi.py
import pytest

from hbayesdm.models import bart_prospect_phi


def test_bart_prospect_phi():
    _ = bart_prospect_phi(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_prospect_phi.R
context("Test bart_prospect_phi")
library(hBayesDM)

test_that("Test bart_prospect_phi", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_prospect_phi(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
